Asterix/optics/coronagraph.py

```diff
@@ -587,7 +588,7 @@ def ClassicalLyot(self):
         """
 
         rad_LyotFP_pix = self.rad_lyot_fpm * self.Lyot_fpm_sampling
-        ClassicalLyotFPM = 1. - phase_ampl.roundpupil(self.dim_fpm, rad_LyotFP_pix)
+        ClassicalLyotFPM = 1. - phase_ampl.roundpupil(self.dim_fpm, rad_LyotFP_pix, grey_pup_bin_factor=1)
```

Comment:
default can change. Tet's add it when we're sure it's not what we want. Here is very specific because this is the only time we actually use roundpupil not as a pupil but in focal plane, which is a bit of a stretch